src/mlpack/methods/quic_svd/quic_svd.hpp

```diff
@@ -94,4 +97,20 @@ class QUIC_SVD
 } // namespace svd
 } // namespace mlpack
 
+namespace mlpack {
+namespace cf {
+
+//! Factorizer traits of Quic SVD.
+template<>
+class FactorizerTraits<mlpack::svd::QUIC_SVD>
+{
+ public:
+  //! Data provided to QuicSVD need not be cleaned.
+  static const bool UsesCoordinateList = true;
```

Comment:
I don't think QUIC_SVD supports coordinate lists---that means that the data is expected as a matrix filled with rows of `(row, col, value)`.  Basically a coordinate list is just another representation of a sparse matrix.